src/heart.c

```diff
@@ -631,30 +631,50 @@ static int heart_cmd_info_reply()
      *  <KEY>=<VALUE> NEWLINE
      *  ...
      */
-    p += sprintf(p, "program_name=" PROGRAM_NAME "\nprogram_version=" PROGRAM_VERSION_STR "\n");
+    p += sprintf(p, "program_name=" PROGRAM_NAME "\nprogram_version=" PROGRAM_VERSION_STR "\nheartbeat_timeout=%d\n", heart_beat_timeout);
 
     ret = ioctl(watchdog_fd, WDIOC_GETSUPPORT, &info);
     if (ret == 0) {
         p += sprintf(p, "identity=%s\n", info.identity);
         p += sprintf(p, "firmware_version=%u\n", info.firmware_version);
-        p += sprintf(p, "options=0x%08x\n", info.options);
+        p += sprintf(p, "options=");
+        if (info.options & WDIOF_OVERHEAT) p += sprintf(p, "overheat,");
+        if (info.options & WDIOF_FANFAULT) p += sprintf(p, "fanfault,");
```

Comment:
```suggestion
        if (info.options & WDIOF_FANFAULT) p += sprintf(p, "fan_fault,");
```
Is it one word or two... 😉

Reply:
lol. I gave up here and unused the Linux names.